setup/launch.py

```
@@ -409,7 +384,24 @@ def activate_conda_env(env_name: str = None) -> bool:
 
     if not is_conda_available():
         logger.debug("Conda not available, skipping environment activation.")
-=======
+        return False
+
+    conda_info = get_conda_info()
```

Flake8 (7.3.0)

[error] undefined name 'get_conda_info'

(F821)